Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridViewButtonCell.cs

```diff
@@ -143,15 +143,24 @@ public ButtonStyle ButtonStyle
                 // Update the display text
                 if ((kDGV.Columns[ColumnIndex] is KryptonDataGridViewButtonColumn { UseColumnTextForButtonValue: true } col) && !kDGV.Rows[rowIndex].IsNewRow)
                 {
-                    _shortTextValue.ShortText = col.Text;
+                    if (_shortTextValue != null)
+                    {
+                        _shortTextValue.ShortText = col.Text;
+                    }
                 }
-                else if (!string.IsNullOrEmpty(FormattedValue?.ToString()))
+                else if (FormattedValue != null ? !string.IsNullOrEmpty(FormattedValue.ToString()) : false)
```

Comment:
Not sure this change adds anything, a null check is done in the string checker ??

Reply:
Reverted